src/components/UI/Text/InputNumber.vue

```diff
@@ -257,7 +262,14 @@ export default {
     },
     keyPress(event) {
       // invoked when a key is pressed
-      if (this.maxLength != null && this.localValue.length == this.maxLength) {
+      // character key not accepted
+      event = event || window.event;
+      var charCode = (typeof event.which == "undefined") ? event.keyCode : event.which;
+      var charStr = String.fromCharCode(charCode);
```

Comment:
```suggestion
      let charCode = (typeof event.which == "undefined") ? event.keyCode : event.which;
      let charStr = String.fromCharCode(charCode);
```

We should be using `let` here instead of `var`
You can read up on which one to use when

Reply:
let allows us to declare variables that are limited in scope to the block, statement, or expression on which it is used. 
var keyword is used to define a variable globally, or locally to an entire function regardless of block scope.
Will use let here.